src/Pages/suyeonkim/Main/Feeds/Feeds.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Feed from './Feed/Feed';
+import './Feeds.scss';
+
+class Feeds extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      feeds: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/suyeonkim/feed.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(feedInfo => {
+        this.setState({
+          feeds: feedInfo,
+        });
+      });
+  }
+  render() {
```

Comment:
```suggestion
  componentDidMount() {
    fetch('/data/suyeonkim/feed.json')
      .then(res => res.json())
      .then(feeds => this.setState({ feeds });
    }
  }
  
  render() {
```
- public에서 접근하는 경우 url과 포트번호를 생략할 수 있습니다
- fetch api의 기본 method는 get이기 때문에 역시 생략 가능합니다
- 객체의 키와 value의 변수명이 같을 경우 생략해줄 수 있습니다 (객체 단축 속성명)